### DISCUSSION TITLE
Replication of Workflow

### DISCUSSION THREAD

**Discussion by Tandoy** [apache/dolphinscheduler#8231] [Q&A] [answered]
Currently, only workflow that is not online can be copied. This often leads to forgetting to go online and schedule again after the offline workflow is copied.

![微信截图_20220128094656](https://user-images.githubusercontent.com/56899730/151472977-b37a32c6-7ab0-4173-add1-feed37da4f49.png)

**Comment by davidzollo** [comment, marked as answer]
good suggestion,  would you like to submit a PR to improve this Feature?

**Reply by Tandoy** [reply]
got it,i will try
